src/main/java/jp/ne/paypay/api/WalletApi.java

```diff
@@ -48,11 +49,9 @@ public void setApiClient(ApiClient apiClient) {
      * @throws ApiException If fail to serialize the request body object
```

Comment:
update the javadoc to mention that this is for both check-wallet-balance and get-balance